docs/user-guide/features.rst

```diff
@@ -33,6 +33,27 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Boolean dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Pulp supports RPM content with `boolean dependencies
+<http://rpm.org/user_doc/boolean_dependencies.html>`_ in the ``Requires`` field
+in these basic repository and content management use cases:
+
+* synchronization, publication and content upload
+
+* copying content between repositories
+
+* displaying boolean dependencies in the ``Requires`` field
+
+* providing the content to the ``dnf`` client to process boolean dependencies
+
+.. Note::
+
+  Pulp actually processing boolean dependencies is a matter of future features.
```

Comment:
this sounds to me like master Yoda talking :dancer:

Reply:
Agreed. Probably should be:

"Pulp doesn't currently process boolean dependencies but plans to do so in the future."

Reply:
/me master, aww :D

updating